### PR TITLE
Adjust README to provide workaround with Angular material issue

### PR DESCRIPTION
**Pull request by AhnpGit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +119,19 @@ exports[`CalcComponent should snap 1`] = `
 
 Problems may arise if you're using custom builds (this preset is tailored for `angular-cli` as firsty priority). Please be adivsed that every entry in default configuration may be overriden to best suite your app's needs.
 
+Angular might have problem with Jest regarding to JSDOM missing transform property with Angular material, there is a work around for it. Add this to your `jestGlobalMocks` file
```
Comment (thymikee) [inline README.md]: Can we add a title like `The animation trigger "transformMenu" has failed ` or so like in other cases?
typos: 
* "Angular material" -> "Angular Material"
* "work around" -> "workaround"

Comment (AhnpGit) [reply README.md]: done :)

File: README.md [hunk 2]
```diff
@@ -119,6 +119,19 @@ exports[`CalcComponent should snap 1`] = `
 
 Problems may arise if you're using custom builds (this preset is tailored for `angular-cli` as firsty priority). Please be adivsed that every entry in default configuration may be overriden to best suite your app's needs.
 
+The animation trigger "transformMenu" has failed with Angular Material, there is a workaround for it. Add this to your `jestGlobalMocks` file
```
Comment (thymikee) [inline README.md]: What I meant was:
```
### The animation trigger "transformMenu" has failed

<<Something something about JSDOM>>
Add this to your `jestGlobalMocks` file:
```

Comment (AhnpGit) [reply README.md]: oh I got it now

**Comment by thymikee** [comment]
Thank you!

**Pull request merged**